sa/model_test.go

```diff
@@ -344,37 +344,27 @@ func insertCertificate(ctx context.Context, dbMap *db.WrappedMap, fc clock.FakeC
 		SerialNumber: serialBigInt,
 	}
 
-	testKey := makeKey()
```

Comment:
Seeing this removal makes me think we could use an improved `makeKey()` that runs `ecdsa.GenerateKey` and panics on error. Could save us a few lines of boilerplate here and there.

Reply:
Yeah, makes sense, but I think not as part of this (already large) PR.